index.html

```diff
@@ -42,3 +42,110 @@ <h1 class="hero-title">
         
 </body>
 </html>
+
+<!-- My Recent Work Section -->
+<!-- work section -->
+<section id="recent_work" class="center-content">
+    <h1 class="primary-font-style">
+        <b>My Recent <br> Works</b></h1>
+    <div class="indicator"></div>
+    
+    <div class="careers_list">
+    
+        <div class="careers-card">
+        <img src="Images/img-placeholder.png" alt="placeholder">
+        <div class="careers-card-footer">
+            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
+            <ul class="tags">
+                <li class="Lang-tag">Ruby on rails</li>
+                <li class="Lang-tag">Css</li>
+                <li class="Lang-tag">Javascript</li>
+                <li class="Lang-tag">Html</li>
+            </ul> 
+            <div class="action">
+                <button class="btn-primary primary-font-style">See project</button>
```

Comment:
Great job @187jjay187 🤝

- [ ] Kindly implement all button interactions of `hover`, `active`, and `pressed` across all your buttons in the design.